RussianFilter/sections/specific.txt

```diff
@@ -34,7 +34,8 @@ brestcity.com##.BlockTop1
 brestcity.com###page > div#wrapper~a[rel="nofollow"] > img
 mysitem.ru##.entry-content > .custom
 pornotor.org##center > a[rel="nofollow"] > img
-kompkimi.ru##.widget_sitemoney
+||kompkimi.ru/wp-content/uploads/*/direct.php
+||info.kinoclub77.ru^$domain=kompkimi.ru
```

Comment:
remove `$domain=kompkimi.ru` and move rule to adservers_firstparty.txt

Reply:
@Alex-302, done